core/src/main/java/org/infinispan/manager/DefaultCacheManager.java

```diff
@@ -744,6 +754,7 @@ private void internalStart(boolean block) {
       try {
          globalComponentRegistry.getComponent(CacheManagerJmxRegistration.class).start();
          globalComponentRegistry.start();
+         configurationManager.removeInvalids();
```

Comment:
How will the users be able to correct the configuration if we remove it and they can't see it in the console?

Reply:
the users can't see the console if there is an invalid configuration stored :)